docs/Acceptable-Casks.md

```diff
@@ -126,3 +126,7 @@ Common reasons to reject a Cask entirely:
 Common reasons to reject a Cask from the main repo:
 
 + The cask was submitted to the wrong repo. When drafting a cask, consult “[Finding a Home For Your Cask](#finding-a-home-for-your-cask)” to see where it belongs.
+
+## No cask is guaranteed to be accepted
+
+Follow the guidelines above and your submission has a great chance of being accepted. But remember documentation tends to lag behind current decision-making and we can’t predict every case. Maitainers may override these rules when experience tells us it will lead to a better overall Homebrew.
```

Comment:
small typo!
```suggestion
Follow the guidelines above and your submission has a great chance of being accepted. But remember documentation tends to lag behind current decision-making and we can’t predict every case. Maintainers may override these rules when experience tells us it will lead to a better overall Homebrew.
```